Watching for file changes with StatReloader
INFO 2023-08-27 18:30:32,021 autoreload 5788 9804 Watching for file changes with StatReloader
INFO 2023-08-27 18:30:35,733 views 5788 9856 Index page accessed
INFO 2023-08-27 18:30:35,734 basehttp 5788 9856 "GET / HTTP/1.1" 200 223
INFO 2023-08-27 18:30:36,889 views 5788 9856 About page accessed
INFO 2023-08-27 18:30:36,890 basehttp 5788 9856 "GET /about/ HTTP/1.1" 200 279
<<<<<<< HEAD
INFO 2023-09-09 21:47:20,406 autoreload 8372 18112 Watching for file changes with StatReloader
INFO 2023-09-09 21:47:22,267 views 8372 7976 Index page accessed
INFO 2023-09-09 21:47:22,268 basehttp 8372 7976 "GET / HTTP/1.1" 200 223
WARNING 2023-09-09 21:47:22,448 log 8372 7976 Not Found: /favicon.ico
WARNING 2023-09-09 21:47:22,448 basehttp 8372 7976 "GET /favicon.ico HTTP/1.1" 404 2548
WARNING 2023-09-09 21:47:34,740 log 8372 7976 Not Found: /hw2/1
WARNING 2023-09-09 21:47:34,741 basehttp 8372 7976 "GET /hw2/1 HTTP/1.1" 404 2638
INFO 2023-09-09 21:47:49,932 basehttp 8372 7976 "GET /hw2/change_product/1 HTTP/1.1" 301 0
INFO 2023-09-09 21:47:49,997 basehttp 8372 17420 "GET /hw2/change_product/1/ HTTP/1.1" 200 1423
ERROR 2023-09-09 21:48:34,619 log 8372 17420 Internal Server Error: /hw2/change_product/1/
Traceback (most recent call last):
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "G:\Домашнее задание GeekBrains\pythonProject\hw_1_project\myapp_hw2\views.py", line 12, in product_form
    pk=Product.objects.filter(product_id).first(),
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\db\models\query.py", line 1436, in filter
    return self._filter_or_exclude(False, args, kwargs)
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\db\models\query.py", line 1454, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\db\models\query.py", line 1461, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\db\models\sql\query.py", line 1545, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\db\models\sql\query.py", line 1576, in _add_q
    child_clause, needed_inner = self.build_filter(
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\db\models\sql\query.py", line 1423, in build_filter
    arg, value = filter_expr
TypeError: cannot unpack non-iterable int object
ERROR 2023-09-09 21:48:34,622 basehttp 8372 17420 "POST /hw2/change_product/1/ HTTP/1.1" 500 100822
INFO 2023-09-09 21:49:32,791 autoreload 8372 18112 G:\Домашнее задание GeekBrains\pythonProject\hw_1_project\myapp_hw2\views.py changed, reloading.
INFO 2023-09-09 21:49:33,494 autoreload 280 8580 Watching for file changes with StatReloader
INFO 2023-09-09 21:49:34,046 basehttp 280 13400 "POST /hw2/change_product/1/ HTTP/1.1" 200 1572
INFO 2023-09-09 21:52:03,707 autoreload 280 8580 G:\Домашнее задание GeekBrains\pythonProject\hw_1_project\myapp_hw2\models.py changed, reloading.
INFO 2023-09-09 21:52:04,561 autoreload 4984 2312 Watching for file changes with StatReloader
INFO 2023-09-09 21:59:02,572 autoreload 4984 2312 G:\Домашнее задание GeekBrains\pythonProject\hw_1_project\myapp_hw2\views.py changed, reloading.
INFO 2023-09-09 21:59:03,170 autoreload 13372 17148 Watching for file changes with StatReloader
INFO 2023-09-09 22:00:01,480 autoreload 11544 14864 Watching for file changes with StatReloader
INFO 2023-09-09 22:01:09,422 autoreload 11544 14864 G:\Домашнее задание GeekBrains\pythonProject\hw_1_project\myapp_hw2\urls.py changed, reloading.
INFO 2023-09-09 22:01:10,027 autoreload 5648 11196 Watching for file changes with StatReloader
INFO 2023-09-09 22:01:16,420 basehttp 5648 5772 "GET /hw2/upload_image_to_product HTTP/1.1" 301 0
INFO 2023-09-09 22:01:16,438 basehttp 5648 17232 "GET /hw2/upload_image_to_product/ HTTP/1.1" 200 631
INFO 2023-09-09 22:01:37,927 autoreload 5648 11196 G:\Домашнее задание GeekBrains\pythonProject\hw_1_project\myapp_hw2\forms.py changed, reloading.
INFO 2023-09-09 22:08:42,011 autoreload 1976 5736 Watching for file changes with StatReloader
INFO 2023-09-09 22:10:12,776 basehttp 1976 8924 "GET /hw2/upload_image_to_product/ HTTP/1.1" 200 646
INFO 2023-09-09 22:11:46,767 autoreload 1976 5736 G:\Домашнее задание GeekBrains\pythonProject\hw_1_project\myapp_hw2\views.py changed, reloading.
INFO 2023-09-09 22:11:47,496 autoreload 17136 5344 Watching for file changes with StatReloader
INFO 2023-09-09 22:13:07,252 autoreload 17136 5344 G:\Домашнее задание GeekBrains\pythonProject\hw_1_project\myapp_hw2\views.py changed, reloading.
INFO 2023-09-09 22:13:07,889 autoreload 15876 2840 Watching for file changes with StatReloader
ERROR 2023-09-09 22:13:13,787 log 15876 12060 Internal Server Error: /hw2/upload_image_to_product/
Traceback (most recent call last):
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: upload_image_to_product() missing 1 required positional argument: 'product_id'
ERROR 2023-09-09 22:13:13,788 basehttp 15876 12060 "GET /hw2/upload_image_to_product/ HTTP/1.1" 500 62782
ERROR 2023-09-09 22:13:20,427 log 15876 12060 Internal Server Error: /hw2/upload_image_to_product/
Traceback (most recent call last):
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: upload_image_to_product() missing 1 required positional argument: 'product_id'
ERROR 2023-09-09 22:13:20,428 basehttp 15876 12060 "GET /hw2/upload_image_to_product/ HTTP/1.1" 500 62782
INFO 2023-09-09 22:13:33,413 autoreload 15876 2840 G:\Домашнее задание GeekBrains\pythonProject\hw_1_project\myapp_hw2\urls.py changed, reloading.
INFO 2023-09-09 22:13:34,053 autoreload 1840 10688 Watching for file changes with StatReloader
WARNING 2023-09-09 22:13:42,612 log 1840 13392 Not Found: /hw2/upload_image_to_product/
WARNING 2023-09-09 22:13:42,612 basehttp 1840 13392 "GET /hw2/upload_image_to_product/ HTTP/1.1" 404 2937
INFO 2023-09-09 22:13:51,621 basehttp 1840 13392 "GET /hw2/upload_image_to_product/1 HTTP/1.1" 301 0
INFO 2023-09-09 22:13:51,636 basehttp 1840 2584 "GET /hw2/upload_image_to_product/1/ HTTP/1.1" 200 605
ERROR 2023-09-09 22:13:59,056 log 1840 2584 Internal Server Error: /hw2/upload_image_to_product/1/
Traceback (most recent call last):
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "G:\Домашнее задание GeekBrains\pythonProject\hw_1_project\myapp_hw2\views.py", line 33, in upload_image_to_product
    image=form.cleaned_data['image']
KeyError: 'image'
ERROR 2023-09-09 22:13:59,059 basehttp 1840 2584 "POST /hw2/upload_image_to_product/1/ HTTP/1.1" 500 67937
INFO 2023-09-09 22:14:33,743 autoreload 1840 10688 G:\Домашнее задание GeekBrains\pythonProject\hw_1_project\myapp_hw2\views.py changed, reloading.
INFO 2023-09-09 22:14:34,423 autoreload 4424 13260 Watching for file changes with StatReloader
ERROR 2023-09-09 22:14:39,279 log 4424 4976 Internal Server Error: /hw2/upload_image_to_product/1/
Traceback (most recent call last):
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "G:\Домашнее задание GeekBrains\pythonProject\hw_1_project\myapp_hw2\views.py", line 33, in upload_image_to_product
    photo=form.cleaned_data['image']
KeyError: 'image'
ERROR 2023-09-09 22:14:39,279 basehttp 4424 4976 "POST /hw2/upload_image_to_product/1/ HTTP/1.1" 500 67937
ERROR 2023-09-09 22:14:45,019 log 4424 4976 Internal Server Error: /hw2/upload_image_to_product/1/
Traceback (most recent call last):
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "G:\Домашнее задание GeekBrains\pythonProject\hw_1_project\myapp_hw2\views.py", line 33, in upload_image_to_product
    photo=form.cleaned_data['image']
KeyError: 'image'
ERROR 2023-09-09 22:14:45,021 basehttp 4424 4976 "POST /hw2/upload_image_to_product/1/ HTTP/1.1" 500 67937
INFO 2023-09-09 22:15:35,623 autoreload 4424 13260 G:\Домашнее задание GeekBrains\pythonProject\hw_1_project\myapp_hw2\views.py changed, reloading.
INFO 2023-09-09 22:15:36,283 autoreload 1016 7852 Watching for file changes with StatReloader
ERROR 2023-09-09 22:15:40,593 log 1016 8080 Internal Server Error: /hw2/upload_image_to_product/1/
Traceback (most recent call last):
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\db\backends\utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return super().execute(query, params)
sqlite3.IntegrityError: NOT NULL constraint failed: myapp_hw2_product.price

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "G:\Домашнее задание GeekBrains\pythonProject\hw_1_project\myapp_hw2\views.py", line 35, in upload_image_to_product
    product.save()
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\db\models\base.py", line 814, in save
    self.save_base(
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\db\models\base.py", line 877, in save_base
    updated = self._save_table(
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\db\models\base.py", line 990, in _save_table
    updated = self._do_update(
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\db\models\base.py", line 1054, in _do_update
    return filtered._update(values) > 0
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\db\models\query.py", line 1231, in _update
    return query.get_compiler(self.db).execute_sql(CURSOR)
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\db\models\sql\compiler.py", line 1984, in execute_sql
    cursor = super().execute_sql(result_type)
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\db\models\sql\compiler.py", line 1562, in execute_sql
    cursor.execute(sql, params)
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\db\backends\utils.py", line 102, in execute
    return super().execute(sql, params)
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\db\backends\utils.py", line 67, in execute
    return self._execute_with_wrappers(
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\db\backends\utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\db\backends\utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\db\backends\utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "G:\Домашнее задание GeekBrains\Django(HW)\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return super().execute(query, params)
django.db.utils.IntegrityError: NOT NULL constraint failed: myapp_hw2_product.price
ERROR 2023-09-09 22:15:40,593 basehttp 1016 8080 "POST /hw2/upload_image_to_product/1/ HTTP/1.1" 500 149435
INFO 2023-09-09 22:18:58,493 autoreload 1016 7852 G:\Домашнее задание GeekBrains\pythonProject\hw_1_project\myapp_hw2\views.py changed, reloading.
INFO 2023-09-09 22:18:59,123 autoreload 9396 2776 Watching for file changes with StatReloader
INFO 2023-09-09 22:19:03,798 basehttp 9396 12508 "POST /hw2/upload_image_to_product/1/ HTTP/1.1" 200 646
=======
INFO 2023-09-03 18:20:10,224 autoreload 13644 6372 Watching for file changes with StatReloader
INFO 2023-09-03 18:20:12,529 views 13644 17712 Index page accessed
INFO 2023-09-03 18:20:12,531 basehttp 13644 17712 "GET / HTTP/1.1" 200 223
WARNING 2023-09-03 18:20:12,707 log 13644 17712 Not Found: /favicon.ico
WARNING 2023-09-03 18:20:12,707 basehttp 13644 17712 "GET /favicon.ico HTTP/1.1" 404 2444
WARNING 2023-09-03 18:20:34,853 log 13644 17712 Not Found: /sem3/posts/2
WARNING 2023-09-03 18:20:34,854 basehttp 13644 17712 "GET /sem3/posts/2 HTTP/1.1" 404 2447
INFO 2023-09-03 18:21:40,506 autoreload 13644 6372 G:\Домашнее задание GeekBrains\pythonProject\hw_1_project\hw_1_project\urls.py changed, reloading.
INFO 2023-09-03 18:21:41,242 autoreload 16476 8004 Watching for file changes with StatReloader
INFO 2023-09-03 18:21:43,831 basehttp 16476 14156 "GET /sem3/posts/2 HTTP/1.1" 200 176
INFO 2023-09-03 18:21:46,686 basehttp 16476 14156 "GET /sem3/post/2 HTTP/1.1" 200 1586
INFO 2023-09-03 18:22:11,814 basehttp 16476 14156 "GET /sem3/posts/4 HTTP/1.1" 200 176
INFO 2023-09-03 18:22:17,582 basehttp 16476 14156 "GET /sem3/posts/9 HTTP/1.1" 200 177
INFO 2023-09-03 18:22:22,735 basehttp 16476 14156 "GET /sem3/posts/1 HTTP/1.1" 200 156
INFO 2023-09-03 18:22:30,678 basehttp 16476 14156 "GET /sem3/posts/10 HTTP/1.1" 200 297
INFO 2023-09-03 18:22:35,080 basehttp 16476 14156 "GET /sem3/post/24 HTTP/1.1" 200 1304
INFO 2023-09-03 18:39:45,000 autoreload 15980 17852 Watching for file changes with StatReloader
INFO 2023-09-03 18:41:39,750 autoreload 9296 2032 Watching for file changes with StatReloader
WARNING 2023-09-03 18:41:54,398 log 9296 8908 Not Found: /hw3/2
WARNING 2023-09-03 18:41:54,399 basehttp 9296 8908 "GET /hw3/2 HTTP/1.1" 404 2724
INFO 2023-09-03 18:42:06,919 basehttp 9296 8908 "GET /hw3/order/3 HTTP/1.1" 301 0
INFO 2023-09-03 18:42:06,934 basehttp 9296 7024 "GET /hw3/order/3/ HTTP/1.1" 200 269
ERROR 2023-09-03 18:44:27,581 log 9296 7024 Internal Server Error: /hw3/order/3/
Traceback (most recent call last):
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "G:\Домашнее задание GeekBrains\pythonProject\hw_1_project\myapp_hw2\views.py", line 14, in client_orders
    return render(request, 'myapp_hw2/client_orders.html', context)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\defaulttags.py", line 193, in render
    values = list(values)
TypeError: 'ManyRelatedManager' object is not iterable
ERROR 2023-09-03 18:44:27,589 basehttp 9296 7024 "GET /hw3/order/3/ HTTP/1.1" 500 120170
ERROR 2023-09-03 18:47:47,145 log 9296 7024 Internal Server Error: /hw3/order/3/
Traceback (most recent call last):
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "G:\Домашнее задание GeekBrains\pythonProject\hw_1_project\myapp_hw2\views.py", line 14, in client_orders
    return render(request, 'myapp_hw2/client_orders.html', context)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\defaulttags.py", line 193, in render
    values = list(values)
TypeError: 'ManyRelatedManager' object is not iterable
ERROR 2023-09-03 18:47:47,148 basehttp 9296 7024 "GET /hw3/order/3/ HTTP/1.1" 500 120170
INFO 2023-09-03 18:47:56,518 basehttp 9296 7024 "GET /hw3/order/3/ HTTP/1.1" 200 269
ERROR 2023-09-03 18:51:29,461 log 9296 7024 Internal Server Error: /hw3/order/3/
Traceback (most recent call last):
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 505, in parse
    compile_func = self.tags[command]
KeyError: 'endfor'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "G:\Домашнее задание GeekBrains\pythonProject\hw_1_project\myapp_hw2\views.py", line 14, in client_orders
    return render(request, 'myapp_hw2/client_orders.html', context)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 200, in compile_nodelist
    return parser.parse()
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 507, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 568, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 14: 'endfor'. Did you forget to register or load this tag?
ERROR 2023-09-03 18:51:29,466 basehttp 9296 7024 "GET /hw3/order/3/ HTTP/1.1" 500 136254
INFO 2023-09-03 18:52:18,323 basehttp 9296 7024 "GET /hw3/order/3/ HTTP/1.1" 200 325
INFO 2023-09-03 18:52:55,777 basehttp 9296 7024 "GET /hw3/order/3/ HTTP/1.1" 200 325
INFO 2023-09-03 18:52:57,573 basehttp 9296 7024 "GET /hw3/order/3/ HTTP/1.1" 200 439
INFO 2023-09-03 18:53:08,248 basehttp 9296 7024 "GET /hw3/order/1 HTTP/1.1" 301 0
INFO 2023-09-03 18:53:08,258 basehttp 9296 7804 "GET /hw3/order/1/ HTTP/1.1" 200 618
INFO 2023-09-03 18:53:20,415 basehttp 9296 7804 "GET /hw3/order/2 HTTP/1.1" 301 0
INFO 2023-09-03 18:53:20,427 basehttp 9296 628 "GET /hw3/order/2/ HTTP/1.1" 200 610
INFO 2023-09-03 18:53:29,663 basehttp 9296 628 "GET /hw3/order/4 HTTP/1.1" 301 0
INFO 2023-09-03 18:53:29,673 basehttp 9296 18056 "GET /hw3/order/4/ HTTP/1.1" 200 531
INFO 2023-09-03 19:18:33,727 autoreload 16772 10608 Watching for file changes with StatReloader
WARNING 2023-09-03 19:18:48,979 log 16772 3688 Not Found: /hw3/products/week
WARNING 2023-09-03 19:18:48,979 basehttp 16772 3688 "GET /hw3/products/week HTTP/1.1" 404 2972
ERROR 2023-09-03 19:19:00,675 log 16772 3688 Internal Server Error: /hw3/products/2/week
Traceback (most recent call last):
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: client_orders() got an unexpected keyword argument 'date'
ERROR 2023-09-03 19:19:00,676 basehttp 16772 3688 "GET /hw3/products/2/week HTTP/1.1" 500 63911
INFO 2023-09-03 19:20:04,419 autoreload 16772 10608 G:\Домашнее задание GeekBrains\pythonProject\hw_1_project\myapp_hw2\views.py changed, reloading.
INFO 2023-09-03 19:20:05,133 autoreload 9720 4948 Watching for file changes with StatReloader
ERROR 2023-09-03 19:20:05,610 log 9720 15228 Internal Server Error: /hw3/products/2/week
Traceback (most recent call last):
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: client_orders() got an unexpected keyword argument 'date'
ERROR 2023-09-03 19:20:05,612 basehttp 9720 15228 "GET /hw3/products/2/week HTTP/1.1" 500 64048
ERROR 2023-09-03 19:20:06,946 log 9720 15228 Internal Server Error: /hw3/products/2/week
Traceback (most recent call last):
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: client_orders() got an unexpected keyword argument 'date'
ERROR 2023-09-03 19:20:06,948 basehttp 9720 15228 "GET /hw3/products/2/week HTTP/1.1" 500 64048
INFO 2023-09-03 19:20:18,197 autoreload 6160 5608 Watching for file changes with StatReloader
ERROR 2023-09-03 19:20:24,432 log 6160 14432 Internal Server Error: /hw3/products/2/week
Traceback (most recent call last):
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: client_orders() got an unexpected keyword argument 'date'
ERROR 2023-09-03 19:20:24,433 basehttp 6160 14432 "GET /hw3/products/2/week HTTP/1.1" 500 64048
ERROR 2023-09-03 19:20:34,943 log 6160 14432 Internal Server Error: /hw3/products/2/week
Traceback (most recent call last):
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: client_orders() got an unexpected keyword argument 'date'
ERROR 2023-09-03 19:20:34,944 basehttp 6160 14432 "GET /hw3/products/2/week HTTP/1.1" 500 63911
INFO 2023-09-03 19:20:56,712 autoreload 6160 5608 G:\Домашнее задание GeekBrains\pythonProject\hw_1_project\myapp_hw2\urls.py changed, reloading.
INFO 2023-09-03 19:20:57,398 autoreload 720 16200 Watching for file changes with StatReloader
INFO 2023-09-03 19:21:25,743 autoreload 6604 9308 Watching for file changes with StatReloader
INFO 2023-09-03 19:21:33,659 autoreload 6604 9308 G:\Домашнее задание GeekBrains\pythonProject\hw_1_project\myapp_hw2\urls.py changed, reloading.
INFO 2023-09-03 19:21:34,300 autoreload 17968 9428 Watching for file changes with StatReloader
INFO 2023-09-03 19:21:36,098 basehttp 17968 18304 "GET /hw3/products/2/week HTTP/1.1" 200 276
INFO 2023-09-03 19:22:06,002 basehttp 17968 18304 "GET /hw3/products/2/week HTTP/1.1" 200 276
INFO 2023-09-03 19:22:07,379 basehttp 17968 18304 "GET /hw3/products/2/week HTTP/1.1" 200 276
INFO 2023-09-03 19:22:28,761 autoreload 17968 9428 G:\Домашнее задание GeekBrains\pythonProject\hw_1_project\myapp_hw2\views.py changed, reloading.
INFO 2023-09-03 19:22:29,392 autoreload 8988 16724 Watching for file changes with StatReloader
INFO 2023-09-03 19:22:30,563 basehttp 8988 14992 "GET /hw3/products/2/week HTTP/1.1" 200 276
INFO 2023-09-03 19:23:16,931 basehttp 8988 14992 "GET /hw3/products/2/week HTTP/1.1" 200 276
INFO 2023-09-03 19:23:20,024 basehttp 8988 14992 "GET /hw3/products/2/week HTTP/1.1" 200 428
ERROR 2023-09-03 19:25:11,498 log 8988 14992 Internal Server Error: /hw3/products/2/week
Traceback (most recent call last):
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 505, in parse
    compile_func = self.tags[command]
KeyError: 'endif'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "G:\Домашнее задание GeekBrains\pythonProject\hw_1_project\myapp_hw2\views.py", line 38, in client_products
    return render(request, 'myapp_hw2/client_products.html', context)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 200, in compile_nodelist
    return parser.parse()
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\defaulttags.py", line 953, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\defaulttags.py", line 853, in do_for
    nodelist_loop = parser.parse(
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 507, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 558, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 16: 'endif', expected 'empty' or 'endfor'. Did you forget to register or load this tag?
ERROR 2023-09-03 19:25:11,503 basehttp 8988 14992 "GET /hw3/products/2/week HTTP/1.1" 500 170115
INFO 2023-09-03 19:25:54,426 basehttp 8988 14992 "GET /hw3/products/2/week HTTP/1.1" 200 188
INFO 2023-09-03 19:26:26,024 basehttp 8988 14992 "GET /hw3/order/4/ HTTP/1.1" 200 531
INFO 2023-09-03 19:26:27,203 basehttp 8988 14992 "GET /hw3/order/2/ HTTP/1.1" 200 610
INFO 2023-09-03 19:27:10,760 basehttp 8988 14992 "GET /hw3/products/2/week HTTP/1.1" 200 892
ERROR 2023-09-03 19:32:28,280 log 8988 3676 Internal Server Error: /hw3/products/2/week
Traceback (most recent call last):
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\smartif.py", line 179, in translate_token
    op = OPERATORS[token]
KeyError: "date:date:'d M Y'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "G:\Домашнее задание GeekBrains\pythonProject\hw_1_project\myapp_hw2\views.py", line 38, in client_products
    return render(request, 'myapp_hw2/client_products.html', context)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 200, in compile_nodelist
    return parser.parse()
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\defaulttags.py", line 853, in do_for
    nodelist_loop = parser.parse(
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\defaulttags.py", line 952, in do_if
    condition = TemplateIfParser(parser, bits).parse()
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\defaulttags.py", line 885, in __init__
    super().__init__(*args, **kwargs)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\smartif.py", line 170, in __init__
    mapped_tokens.append(self.translate_token(token))
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\smartif.py", line 181, in translate_token
    return self.create_var(token)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\defaulttags.py", line 888, in create_var
    return TemplateLiteral(self.template_parser.compile_filter(value), value)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 600, in compile_filter
    return FilterExpression(token, self)
  File "G:\Домашнее задание GeekBrains\pythonProject\venv\lib\site-packages\django\template\base.py", line 703, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: ':date:'d M Y'' from 'date:date:'d M Y''
ERROR 2023-09-03 19:32:28,285 basehttp 8988 3676 "GET /hw3/products/2/week HTTP/1.1" 500 190552
INFO 2023-09-03 19:32:52,484 basehttp 8988 3676 "GET /hw3/products/2/week HTTP/1.1" 200 892
INFO 2023-09-03 19:33:55,866 basehttp 8988 3676 "GET /hw3/products/2/week HTTP/1.1" 200 812
INFO 2023-09-03 19:41:04,638 basehttp 8988 18080 "GET /hw3/products/2/week HTTP/1.1" 200 812
INFO 2023-09-03 19:41:19,314 autoreload 8988 16724 G:\Домашнее задание GeekBrains\pythonProject\hw_1_project\myapp_hw2\views.py changed, reloading.
INFO 2023-09-03 19:41:20,020 autoreload 2352 12460 Watching for file changes with StatReloader
INFO 2023-09-03 19:41:20,175 basehttp 2352 5600 "GET /hw3/products/2/week HTTP/1.1" 200 812
INFO 2023-09-03 19:41:22,892 basehttp 2352 5600 "GET /hw3/products/2/week HTTP/1.1" 200 812
>>>>>>> 3f27b301cc4ba52a21b966e14e6ccc65d7a3ec5e
INFO 2023-09-12 19:54:44,835 autoreload 2476 12440 Watching for file changes with StatReloader
INFO 2023-09-12 19:55:29,681 autoreload 2476 12440 G:\Домашнее задание GeekBrains\pythonProject\hw_1_project\myapp_hw2\urls.py changed, reloading.
INFO 2023-09-12 19:55:30,311 autoreload 16712 13844 Watching for file changes with StatReloader
INFO 2023-09-12 19:56:00,288 autoreload 16712 13844 G:\Домашнее задание GeekBrains\pythonProject\hw_1_project\myapp_hw2\views.py changed, reloading.
INFO 2023-09-12 19:56:00,919 autoreload 11088 5216 Watching for file changes with StatReloader
INFO 2023-09-12 19:56:12,210 autoreload 11088 5216 G:\Домашнее задание GeekBrains\pythonProject\hw_1_project\myapp_hw2\views.py changed, reloading.
INFO 2023-09-12 19:56:12,865 autoreload 11276 16408 Watching for file changes with StatReloader
INFO 2023-09-12 19:56:14,885 views 11276 12340 Index page accessed
INFO 2023-09-12 19:56:14,887 basehttp 11276 12340 "GET / HTTP/1.1" 200 223
WARNING 2023-09-12 19:56:15,019 log 11276 12340 Not Found: /favicon.ico
WARNING 2023-09-12 19:56:15,020 basehttp 11276 12340 "GET /favicon.ico HTTP/1.1" 404 2757
WARNING 2023-09-12 19:56:57,472 log 11276 12340 Not Found: /order/2
WARNING 2023-09-12 19:56:57,473 basehttp 11276 12340 "GET /order/2 HTTP/1.1" 404 2745
INFO 2023-09-12 19:57:19,336 basehttp 11276 12340 "GET /hw2/order/2 HTTP/1.1" 301 0
INFO 2023-09-12 19:57:19,371 basehttp 11276 4188 "GET /hw2/order/2/ HTTP/1.1" 200 624
INFO 2023-09-12 20:14:11,892 autoreload 11276 16408 G:\Домашнее задание GeekBrains\pythonProject\hw_1_project\myapp_hw2\admin.py changed, reloading.
INFO 2023-09-12 20:14:13,662 autoreload 8216 11388 Watching for file changes with StatReloader
INFO 2023-09-12 20:16:09,496 autoreload 15776 12244 Watching for file changes with StatReloader
INFO 2023-09-12 20:16:14,144 basehttp 15776 9064 "GET /admin/ HTTP/1.1" 302 0
INFO 2023-09-12 20:16:14,218 basehttp 15776 9064 "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4181
INFO 2023-09-12 20:16:16,835 basehttp 15776 9064 "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
INFO 2023-09-12 20:16:16,866 basehttp 15776 9064 "GET /admin/ HTTP/1.1" 200 7083
INFO 2023-09-12 20:16:21,811 basehttp 15776 9064 "GET /admin/myapp_hw2/client/ HTTP/1.1" 200 9970
INFO 2023-09-12 20:16:21,858 basehttp 15776 9064 "GET /admin/jsi18n/ HTTP/1.1" 200 3343
INFO 2023-09-12 20:16:25,197 basehttp 15776 9064 "GET /admin/ HTTP/1.1" 200 7083
INFO 2023-09-12 20:16:26,207 basehttp 15776 9064 "GET /admin/myapp_hw2/order/ HTTP/1.1" 200 12208
INFO 2023-09-12 20:16:26,255 basehttp 15776 9064 "GET /admin/jsi18n/ HTTP/1.1" 200 3343
INFO 2023-09-12 20:16:29,572 basehttp 15776 9064 "GET /admin/ HTTP/1.1" 200 7083
INFO 2023-09-12 20:17:43,956 basehttp 15776 9064 "GET /admin/auth/group/add/ HTTP/1.1" 200 14354
INFO 2023-09-12 20:17:43,996 basehttp 15776 9064 "GET /admin/jsi18n/ HTTP/1.1" 200 3343
INFO 2023-09-12 20:18:40,557 basehttp 15776 9064 "POST /admin/auth/group/add/ HTTP/1.1" 302 0
INFO 2023-09-12 20:18:40,582 basehttp 15776 9064 "GET /admin/auth/group/ HTTP/1.1" 200 9736
INFO 2023-09-12 20:18:40,625 basehttp 15776 9064 "GET /admin/jsi18n/ HTTP/1.1" 200 3343
INFO 2023-09-12 20:18:48,977 basehttp 15776 9064 "GET /admin/auth/group/add/ HTTP/1.1" 200 14354
INFO 2023-09-12 20:18:49,011 basehttp 15776 9064 "GET /admin/jsi18n/ HTTP/1.1" 200 3343
INFO 2023-09-12 20:19:20,664 basehttp 15776 9064 "POST /admin/auth/group/add/ HTTP/1.1" 302 0
INFO 2023-09-12 20:19:20,686 basehttp 15776 9064 "GET /admin/auth/group/ HTTP/1.1" 200 9927
INFO 2023-09-12 20:19:20,737 basehttp 15776 9064 "GET /admin/jsi18n/ HTTP/1.1" 200 3343
INFO 2023-09-12 20:19:25,272 basehttp 15776 9064 "GET /admin/auth/ HTTP/1.1" 200 5576
INFO 2023-09-12 20:19:31,197 basehttp 15776 9064 "GET /admin/auth/user/add/ HTTP/1.1" 200 11163
INFO 2023-09-12 20:19:31,241 basehttp 15776 9064 "GET /admin/jsi18n/ HTTP/1.1" 200 3343
INFO 2023-09-12 20:20:06,301 basehttp 15776 9064 "POST /admin/auth/user/add/ HTTP/1.1" 200 11405
INFO 2023-09-12 20:20:06,344 basehttp 15776 9064 "GET /admin/jsi18n/ HTTP/1.1" 200 3343
INFO 2023-09-12 20:20:48,091 basehttp 15776 9064 "POST /admin/auth/user/add/ HTTP/1.1" 302 0
INFO 2023-09-12 20:20:48,140 basehttp 15776 9064 "GET /admin/auth/user/2/change/ HTTP/1.1" 200 23989
INFO 2023-09-12 20:20:48,210 basehttp 15776 2552 "GET /admin/jsi18n/ HTTP/1.1" 200 3343
INFO 2023-09-12 20:20:48,311 basehttp 15776 9064 "GET /static/admin/js/calendar.js HTTP/1.1" 200 8466
INFO 2023-09-12 20:20:48,331 basehttp 15776 3064 "GET /static/admin/js/admin/DateTimeShortcuts.js HTTP/1.1" 200 19319
INFO 2023-09-12 20:20:48,395 basehttp 15776 9064 "GET /static/admin/img/icon-clock.svg HTTP/1.1" 200 677
INFO 2023-09-12 20:20:48,395 basehttp 15776 3064 "GET /static/admin/img/icon-calendar.svg HTTP/1.1" 200 1086
INFO 2023-09-12 20:21:39,756 basehttp 15776 3064 "POST /admin/auth/user/2/change/ HTTP/1.1" 302 0
INFO 2023-09-12 20:21:39,782 basehttp 15776 3064 "GET /admin/auth/user/ HTTP/1.1" 200 12568
INFO 2023-09-12 20:21:39,819 basehttp 15776 3064 "GET /admin/jsi18n/ HTTP/1.1" 200 3343
INFO 2023-09-12 20:21:39,850 basehttp 15776 3064 "GET /static/admin/img/sorting-icons.svg HTTP/1.1" 200 1097
INFO 2023-09-12 20:21:50,316 basehttp 15776 3064 "GET /admin/auth/user/add/ HTTP/1.1" 200 11163
INFO 2023-09-12 20:21:50,364 basehttp 15776 3064 "GET /admin/jsi18n/ HTTP/1.1" 200 3343
INFO 2023-09-12 20:22:07,718 basehttp 15776 3064 "POST /admin/auth/user/add/ HTTP/1.1" 302 0
INFO 2023-09-12 20:22:07,755 basehttp 15776 3064 "GET /admin/auth/user/3/change/ HTTP/1.1" 200 23929
INFO 2023-09-12 20:22:07,790 basehttp 15776 3064 "GET /admin/jsi18n/ HTTP/1.1" 200 3343
INFO 2023-09-12 20:22:59,243 basehttp 15776 3064 "POST /admin/auth/user/3/change/ HTTP/1.1" 302 0
INFO 2023-09-12 20:22:59,268 basehttp 15776 3064 "GET /admin/auth/user/ HTTP/1.1" 200 12972
INFO 2023-09-12 20:22:59,316 basehttp 15776 3064 "GET /admin/jsi18n/ HTTP/1.1" 200 3343
INFO 2023-09-12 20:23:30,264 basehttp 15776 3064 "GET /admin/ HTTP/1.1" 200 8828
INFO 2023-09-12 20:53:10,684 autoreload 15776 12244 G:\Домашнее задание GeekBrains\pythonProject\hw_1_project\myapp_hw2\admin.py changed, reloading.
INFO 2023-09-12 20:53:11,446 autoreload 2296 12616 Watching for file changes with StatReloader
INFO 2023-09-12 20:53:21,056 basehttp 2296 11476 "GET /admin/myapp_hw2/client/ HTTP/1.1" 200 12519
INFO 2023-09-12 20:53:21,115 basehttp 2296 11476 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-12 20:54:12,023 basehttp 2296 11476 "GET /admin/myapp_hw2/client/3/change/ HTTP/1.1" 200 12147
INFO 2023-09-12 20:54:12,060 basehttp 2296 11476 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-12 20:54:20,954 basehttp 2296 11476 "GET /admin/myapp_hw2/client/ HTTP/1.1" 200 12519
INFO 2023-09-12 20:54:21,568 basehttp 2296 11476 "GET /admin/ HTTP/1.1" 200 9292
INFO 2023-09-12 20:54:32,027 basehttp 2296 11476 "GET /admin/myapp_hw2/order/ HTTP/1.1" 200 15673
INFO 2023-09-12 20:54:32,074 basehttp 2296 11476 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-12 20:54:58,821 basehttp 2296 11476 "GET /admin/myapp_hw2/order/1/change/ HTTP/1.1" 200 11340
INFO 2023-09-12 20:54:58,889 basehttp 2296 11476 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-12 20:55:12,590 basehttp 2296 11476 "GET /admin/myapp_hw2/order/ HTTP/1.1" 200 15673
INFO 2023-09-12 20:55:14,087 basehttp 2296 11476 "GET /admin/ HTTP/1.1" 200 9292
INFO 2023-09-12 20:55:17,417 basehttp 2296 11476 "GET /admin/myapp_hw2/order/ HTTP/1.1" 200 15673
INFO 2023-09-12 20:55:17,464 basehttp 2296 11476 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-12 20:55:18,563 basehttp 2296 11476 "GET /admin/ HTTP/1.1" 200 9292
INFO 2023-09-12 20:55:19,962 basehttp 2296 11476 "GET /admin/myapp_hw2/product/ HTTP/1.1" 200 11776
INFO 2023-09-12 20:55:19,999 basehttp 2296 11476 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-12 20:55:57,021 basehttp 2296 11476 "GET /admin/myapp_hw2/product/3/change/ HTTP/1.1" 200 13111
INFO 2023-09-12 20:55:57,091 basehttp 2296 11964 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-12 20:55:57,163 basehttp 2296 11476 "GET /static/admin/js/collapse.js HTTP/1.1" 200 1803
INFO 2023-09-12 20:56:24,729 basehttp 2296 11476 "GET /admin/myapp_hw2/product/ HTTP/1.1" 200 11776
INFO 2023-09-12 20:56:26,738 basehttp 2296 11476 "GET /admin/myapp_hw2/product/1/change/ HTTP/1.1" 200 13270
INFO 2023-09-12 20:56:26,772 basehttp 2296 11476 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
WARNING 2023-09-12 20:56:33,482 log 2296 11476 Not Found: /kartinki-nyashnye-kotiki-14_aJPhUAG.jpg
WARNING 2023-09-12 20:56:33,483 basehttp 2296 11476 "GET /kartinki-nyashnye-kotiki-14_aJPhUAG.jpg HTTP/1.1" 404 2841
INFO 2023-09-12 20:56:36,263 basehttp 2296 11476 "GET /admin/myapp_hw2/product/1/change/ HTTP/1.1" 200 13270
INFO 2023-09-12 20:56:58,901 basehttp 2296 11476 "GET /admin/myapp_hw2/product/ HTTP/1.1" 200 11776
INFO 2023-09-12 20:56:59,842 basehttp 2296 11476 "GET /admin/ HTTP/1.1" 200 9292
INFO 2023-09-12 20:57:03,620 basehttp 2296 11476 "POST /admin/logout/ HTTP/1.1" 200 3709
INFO 2023-09-12 20:57:05,371 basehttp 2296 11476 "GET /admin/ HTTP/1.1" 302 0
INFO 2023-09-12 20:57:05,385 basehttp 2296 11476 "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4267
INFO 2023-09-12 20:57:33,892 basehttp 2296 11476 "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
INFO 2023-09-12 20:57:33,910 basehttp 2296 11476 "GET /admin/ HTTP/1.1" 200 6130
INFO 2023-09-12 20:57:33,954 basehttp 2296 11476 "GET /static/admin/img/icon-viewlink.svg HTTP/1.1" 200 581
INFO 2023-09-12 20:57:43,909 basehttp 2296 11476 "GET /admin/myapp_hw2/product/ HTTP/1.1" 200 10625
INFO 2023-09-12 20:57:43,955 basehttp 2296 11476 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-12 20:57:46,120 basehttp 2296 11476 "GET /admin/myapp_hw2/product/3/change/ HTTP/1.1" 200 11954
INFO 2023-09-12 20:57:46,153 basehttp 2296 11476 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-12 20:57:50,830 basehttp 2296 11476 "GET /admin/myapp_hw2/product/ HTTP/1.1" 200 10625
INFO 2023-09-12 20:57:55,276 basehttp 2296 11476 "GET /admin/ HTTP/1.1" 200 6130
INFO 2023-09-12 20:57:57,888 basehttp 2296 11476 "POST /admin/logout/ HTTP/1.1" 200 3709
INFO 2023-09-12 20:57:59,262 basehttp 2296 11476 "GET /admin/ HTTP/1.1" 302 0
INFO 2023-09-12 20:57:59,271 basehttp 2296 11476 "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4267
INFO 2023-09-12 20:58:10,538 basehttp 2296 11476 "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
INFO 2023-09-12 20:58:10,554 basehttp 2296 11476 "GET /admin/ HTTP/1.1" 200 6061
INFO 2023-09-12 20:58:18,447 basehttp 2296 11476 "GET /admin/myapp_hw2/client/ HTTP/1.1" 200 9961
INFO 2023-09-12 20:58:21,144 basehttp 2296 11476 "GET /admin/ HTTP/1.1" 200 6061
INFO 2023-09-12 20:58:26,505 basehttp 2296 11476 "POST /admin/logout/ HTTP/1.1" 200 3709
INFO 2023-09-12 20:58:28,091 basehttp 2296 11476 "GET /admin/ HTTP/1.1" 302 0
INFO 2023-09-12 20:58:28,100 basehttp 2296 11476 "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4267
INFO 2023-09-12 20:58:31,767 basehttp 2296 11476 "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
INFO 2023-09-12 20:58:31,785 basehttp 2296 11476 "GET /admin/ HTTP/1.1" 200 9292
